pom.xml

```diff
@@ -168,7 +178,7 @@
           <plugin>
             <groupId>org.jreleaser</groupId>
             <artifactId>jreleaser-maven-plugin</artifactId>
-            <version>0.9.0</version>
+            <version>1.4.0</version>
```

Comment:
@aalmiray, given this is a major version bump, will this require updates to the JReleaser config, too?

Reply:
I've tried my best to keep backward compatibility as much as possible